apps/judicial-system/api/src/app/modules/case/dto/updateCase.input.ts

```diff
@@ -276,4 +276,8 @@ export class UpdateCaseInput implements UpdateCase {
   @Allow()
   @Field(() => String, { nullable: true })
   readonly subpoenaType?: SubpoenaType
+
+  @Allow()
+  @Field({ nullable: true })
+  readonly defendantRefusesHavingDefender?: boolean
```

Comment:
I think a more accurate name for this is something like `defendantWaivesRightToCounsel`. What do you think @gudjong

Reply:
Agree @oddsson

Reply:
Perfect